doc-content/enterprise-only/cases/case-management-subcases-con.adoc

```diff
@@ -1,11 +1,40 @@
-[id='case-management-subcases-proc_{context}']
-= Creating sub cases in case management projects
+[id='case-management-subcases-con{context}']
+= Subcases
 
-Subcases give you the flexibility to compose advanced cases that consists of other cases. This means that you can split large and complex cases into multiple layers of abstraction and even multiple case projects. This is similar to splitting a process into multiple subprocesses.
+Subcases give you the flexibility to compose advanced cases that consist of other cases. This means that you can split large and complex cases into multiple layers of abstraction and even multiple case projects. This is similar to splitting a process into multiple subprocesses.
 
 A subcase is another case definition that is invoked from within another case instance or a regular process instance. It has all of the capabilities of a regular case instance:
 
 * It has a dedicated case file.
 * It is isolated from any other case instance.
 * It has its own set of case roles.
 * It has its own case prefix.
+
+You can add subcases to your case definition using the process designer. A subcase is another case within your case project, similar to having a subprocess within your process. Subcases can also be added to a regular business process, which enables you to start a case from within a process instance.
+
+The `Sub Case` asset can be found in the case definition process designer *Object Library* under *Cases*:
+
+image::cases/case-management-subcase-designer.png[]
+
+The *Sub Case Data I/O* window supports the following set of input parameters to properly configure and start the subcase:
+
+image::cases/case-management-subcase-dataio.png[]
+
+Independent::
+Optional indicator that tells the {PROCESS_ENGINE} if the case instance is independent. If it is independent, the main case instance will not wait for its completion. This property is `false` by default.
+GroupRole_XXX::
+Optional group to case role mapping. The role names belonging to this case instance can be referenced here, meaning that participants of the main case can be mapped to participants of the subcase. This means that whatever group was assigned to the main case will be automatically assigned to subcase, where `XXX` is the role name and the `value` is the group role assignment.
+DataAccess_XXX::
+Optional data access restrictions where `XXX` is the name of data item and the `value` is the access restrictions.
+DestroyOnAbort::
+Optional indicator that tells the {PROCESS_ENGINE} what to do when the subcase activity is aborted: cancel or destroy the subcase. This defaults to `true`, which destroys the subcase and removes the case file.
```

Comment:
that tells the {PROCESS_ENGINE} whether to cancel or destroy the subcase when the subcase activity is aborted. The default value is true..

Reply:
Done